WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -399,9 +399,19 @@ + (BOOL)keyboardsPreference:(nonnull NSString *)key
   Class controllerClass = NSClassFromString(controllerClassName);
   TIPreferencesController *controller = [controllerClass sharedPreferencesController];
   if ([key isEqualToString:FBKeyboardAutocorrectionKey]) {
-    return [controller boolForPreferenceKey:FBKeyboardAutocorrectionKey];
+    if ([controller respondsToSelector:@selector(boolForPreferenceKey:)]) {
+      return [controller boolForPreferenceKey:FBKeyboardAutocorrectionKey];
+    } else {
+      [FBLogger log:@"Updating keyboard autocorrection preference is not supported"];
+      return nil;
```

Comment:
shoudn't only YES and NO be returned for BOOL?

Reply:
Yeah, i changed enum way to handle the enabled status for keyboard pref.